File: Arrays/Python/DoubleArray.py
# Double every elements in the array
def double_array(array):
    for i in range(len(array)):
        array[i] *= 2

    print("Array elements after doubling: ")
    for i in array:
        print(i, end=" ")
    print()

if __name__ == "__main__":
    array = [1, 2, 3, 4, 5]
    double_array(array)

File: Arrays/C++/DoubleArray.cpp
//Double every element in the array
#include <iostream>
using namespace std;

void DoubleArray(int array[], int size){
    for (int i = 0; i < size; i++){
        array[i] *= 2;
    }

    cout << "Array elements after doubling: " << endl;
    for (int i = 0; i < size; i++){
        cout << array[i] << " ";
    }
}

int main() {
    int array[] = {1, 2, 3, 4, 5};
    int size  = sizeof(array)/sizeof(array[0]);
    DoubleArray(array, size);
    return 0;
}